deps/graph-parser/src/logseq/graph_parser/extract.cljc

```diff
@@ -88,9 +94,10 @@
 
 ;; TODO: performance improvement
 (defn- extract-pages-and-blocks
-  [format ast properties file content {:keys [date-formatter page-name-order db] :as options}]
+  "uri-encoded? - if is true, apply URL decode on the file path"
+  [format ast properties file content {:keys [date-formatter db uri-encoded?] :as options}]
   (try
-    (let [page (get-page-name file ast page-name-order)
+    (let [page (get-page-name file ast uri-encoded?)
```

Comment:
Basically the `:file/path` are urlencoded on mobile platforms since https://github.com/logseq/logseq/pull/6402. 
The urlencoded `:file/path` has no effect for using as a retrieval key in the same client, but here it's used to parse the page title, so decoding is required.

Reply:
good to know